File: week-02/day-02/factorio.py
# - Create a function called `factorio`
#   that returns it's input's factorial

a = int(input("Enter an integer: "))

def factorio(n):
    if n < 0:
        print("Too small, must be 0 or greater!")
        return
    elif n == 0:
        return 1
    else:
        f = 1
        for i in range(1, n + 1):
            f *= i
        return f

print(factorio(a))

File: week-07/day-2/factorio.js
'use strict';
// - Create a function called `factorio`
//   that returns it's input's factorial

function factorio(number){
  let fact = 1;
  for(let i = 2; i <= number; i++){
    fact *= i;
  }
  if(number < 0){
    fact = 'The number must be non negative';
  }
  return fact;
}

console.log(factorio(0));
console.log(factorio(1));
console.log(factorio(5));
console.log(factorio(10));
console.log(factorio(-1));